src/app/layout/Layout.module.css

```diff
@@ -39,6 +39,7 @@
 .sidebar.hide {
     width: 0px;
     height: 0px;
+    visibility: hidden;
```

Comment:
I noticed that when trying to tab around from the header-bar, the sidebaritems stole focus - so this should fix that.